src/braket/pennylane_plugin/ops.py

```diff
@@ -456,3 +456,60 @@ def compute_matrix(phi_0, phi_1):
     def adjoint(self):
         (phi_0, phi_1) = self.parameters
         return MS(phi_0 + np.pi, phi_1, wires=self.wires)
+
+
+class AAMS(Operation):

```

Comment:
What about MolmerSorensen? I've seen a pattern of renaming classes in a backwards-compatible way by creating a new class with a variation of the old one. In this case, it's more illustrative than `AAMS`, which is no longer obviously an MS gate because more attention is drawn to AA than MS.

Reply:
Seems like a much longer name than any of our other gates, so hesitant to break consistency. AAMS was run by product, but happy to check about MolmerSorensen if you think it's better too